ddtrace/appsec/processor.py

```diff
@@ -270,7 +276,11 @@ def on_span_finish(self, span):
                 data[_Addresses.SERVER_REQUEST_BODY] = body
 
         log.debug("[DDAS-001-00] Executing AppSec In-App WAF with parameters: %s", data)
-        ddwaf_result = self._ddwaf.run(data, self._waf_timeout)  # res is a serialized json
+        ddwaf_result = None
+        try:
+            ddwaf_result = self._ddwaf.run(data, self._waf_timeout)  # res is a serialized json
+        except Exception:
+            log.warning("Error executing Appsec In-App WAF")
```

Comment:
do we want `exc_info=True` here?

do we think there would be concerns of leaking data via this warning log if we include the traceback? would just the exception type be at least enough to make it more useful? (or do the other logs help us out here?)

Reply:
We'll add two checks, one for OSError that should be safe regarding the data leaked, so adding `exc_info=True` in that case, and a second `except Exception` logging only the type of exception.